### PR TITLE
Traitor ai improvements

### PR DESCRIPTION
**Pull request by MacHac**
Traitor AIs now spawn in every game mode, EXCEPT for Revolution and Nuke Ops.

This should raise the chances of traitor AI from 2% up to a far more respectable 15%.

:cl:
tweak: Traitor AIs can now spawn in most game modes, and should be significantly more common.  This is an experimental change, and is subject to removal at any time.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
are you fucking insane

**Comment by FluffySurvivor** [comment]
THANK YOU. 

I haven't seen traitor ai since malf was removed.

File: code/game/gamemodes/blob/blob.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@ var/list/blobs_legit = list() //used for win-score calculations, contains only b
 
 	if(!blob_overminds.len)
 		return 0
-
+	handle_AI_Traitors()
```
Comment (oisin100) [inline code/game/gamemodes/blob/blob.dm]: Ok. Maybe not in blob.

Comment (FluffySurvivor) [inline code/game/gamemodes/blob/blob.dm]: Why not? Blob vs malf time race. It's gonna be fun.

Comment (ShadowDeath6) [inline code/game/gamemodes/blob/blob.dm]: blob vs ai, blob win everytime

**Comment by Super3222** [comment]
Not in blob.

**Comment by Super3222** [comment]
Malf AI during a blob round is crazy.

**Comment by Super3222** [comment]
Malf AI during a blob round is crazy, even if it's a 15% chance not happen. Gang Rounds as well, because they are just as bad as rev rounds.

Traitor, TriatorAb, TraitorAbChan, Changeling, Shadowling, Zombie, All Cult Modes are fine.

**Comment by Super3222** [comment]
Malf AI during a blob round is crazy, even if it's a 15% chance not happen. Gang Rounds as well, because they are just as bad as rev rounds.

**Comment by imtakingabreakdontatme** [comment]
@coiax is allegedly about to finish datum antags, I reccomend waiting on that

**Comment by MacHac** [comment]
@KorPhaeron This was about 10 minutes of code.  When datum antags is ported over, it should wipe it all out, and I'll readd my changes.  On the off chance that it never gets finished or ported or whatever, the changes remain in effect.  That was just my rationale.

@AdamElTablawy Yes.

I'll commit the removal of blob in a little bit.  Gang is fine.

**Comment by ParagonPrime** [comment]
malf AI declaring allegiance to ratvar when :^)

**Comment by Super3222** [comment]
If you could in your changelog, state that this is only a experimental change to see if it'll fit with gameplay.
Because if rounds become too chaotic from this, then we'll have to revert it. They'll need to acknowledge the change to get opinions from them.

**Comment by ghost** [comment]
malf ai is basically ratvar ai anyway

**Comment by ShadowDeath6** [comment]
ratvar a shit, #freeai

File: code/game/gamemodes/blob/blob.dm (hunk 1 above)
Comment (oisin100) [inline code/game/gamemodes/blob/blob.dm]: Welp. We'll see how it goes...

**Pull request merged**